connectors/sources/tests/test_mysql.py

```diff
@@ -188,6 +246,7 @@ async def test_ping_negative(patch_logger):
             await source.ping()
 
 
+@pytest.mark.fail_slow(10)
```

Comment:
Will be fixed with this [PR](https://github.com/elastic/connectors-python/pull/379)